waspc/data/Generator/templates/react-app/src/operations/resources.js

```diff
@@ -0,0 +1,47 @@
+{{= {= =} =}}
+import queryCache from '../queryCache'
+
+
+// Map where key is resource name and value is Set
```

Comment:
resources is actually entities in this case, right?

Reply:
Yes, but I kept it general here since that was easy to do.